configuration/controller/firewalld.sh

```diff
@@ -98,13 +100,19 @@ fi
 
 #---------------------------------------
 
-# Enable HTTPS on public zone
+# Enable required ports on the public zone
 
-if flag_is_set FWD_HTTPS_PUBLIC ; then
-    echo_info "Enabling HTTPS on the public zone"
-    firewall-cmd --zone=public --add-service=https
-    firewall-cmd --permanent --zone=public --add-service=https
-fi
+echo_info "Allowing TCP ports on the public zone"
+
+for range in ${FWD_TCP_PUBLIC[*]}; do
+    firewall-cmd --permanent --zone=public --add-port="$range"/tcp
```

Comment:
Those rules will be loaded when the "firewall-cmd --reload" is executed at the end of the postscript. 
I have kept them in the interface-to-zone section because of the firewalld bug.